db/migrate/20220719180225_add_sort_index_to_school_links.rb

```diff
@@ -0,0 +1,17 @@
+class AddSortIndexToSchoolLinks < ActiveRecord::Migration[6.1]
+  def up
+    add_column :school_links, :sort_index, :integer, default: 0, null: false
+
+    kinds = SchoolLink.distinct.pluck(:kind)
```

Comment:
@developedBySJ : This approach is wrong. The relation above in fact pulls all the schools links from the database and plucks it's `kind`. This is equivalent to `School.all.pluck(:id)`

What we need here is to iterate over every school and select the school links of each school grouped by kind and update the sort index.

Reply:
Fixed in https://github.com/pupilfirst/pupilfirst/commit/24d1d9e559281f029aee4d083e484a89f6b3512f